### ISSUE TITLE
# Train Test Split

### ISSUE THREAD

**Issue by knageswara78** [knageswara78/Python_Solutions#13]
# Train Test Split

from sklearn.model_selection import train_test_split

X_train, X_test, y_train, y_test = train_test_split(df.drop('STATUS',axis=1), 
                                                    df['STATUS'], test_size=0.20, 
                                                    random_state=101)
